backend/app/controllers/spree/admin/refunds_controller.rb

```diff
@@ -8,6 +8,22 @@ class RefundsController < ResourceController
 
       rescue_from Spree::Core::GatewayError, with: :spree_core_gateway_error
 
+      def create
```

Comment:
@jhawthorn @cbrunsdon ?  Are you guys wanting to deprecate `Spree::Backend::Callbacks`?  This seems like the kind of usage they were intended for.